src/app/plugins/dog.plugin.ts

```diff
@@ -29,12 +41,22 @@ export class DogPlugin extends Plugin {
   public async execute(message: IMessage, args?: string[]) {
     let breed = this._parseBreed(args || []);
 
-    if (breed === '' || breed.toLowerCase() === "random") {
+    if (breed === '' || breed.toLowerCase() === 'random') {
       breed = this._breeds[this._getRandomInt(this._breeds.length)];
     }
 
     if (!this._breeds.includes(breed)) {
-      message.reply(`Breeds supported: \n\`\`\`\n${this._breeds.join('\n')}\`\`\``);
+      this.container.messageService
+        .sendTextMessage(
+          message,
+          `Breeds supported: \n\`\`\`\n${this._breeds.join('\n')}\`\`\``,
+          this.messageOptions
+        )
+        .catch((err) => {
+          this.container.loggerService.warn(
+            `Failed to produce the dog breed list. Error info:\n${err}`
+          );
+        });
```

Comment:
We don't have nearly that many breeds

Reply:
I can hope someone volunteers to add to the possible breeds ;-;